to_multi_var

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+ROOT_DIRECTORY="$( dirname "$( readlink -f "${BASH_SOURCE[0]}" )" )"
+readonly ROOT_DIRECTORY
+
+main() {
+    for file in "$@"; do
+        [ -z "${file}" ] && continue
+
+        if [ -f "${file}" ]; then
+            convert_file "${file}"
+        elif [ -d "${file}" ]; then
+            convert_directory "${file}"
+        fi
+    done
+}
+
+convert_file() {
+    local file=$@
+
+    echo "→ Converting var declarations in: ${file}"
+    vim -u "${ROOT_DIRECTORY}/vimrc"                     \
```

Comment:
Yeah, I use that too. But generally I don't like doing this alignments in code. NABD, will merge